topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.md

```diff
@@ -0,0 +1,643 @@
+---
+layout: tutorial_hands_on
+
+title: 'PAPAA PI3K_OG: PanCancer Aberrant Pathway Activity Analysis'
+zenodo_link: https://zenodo.org/record/4306639#.X9FJF-lKgZE
```

Comment:
oh, and it could also be nice to add a `tags:` metadata item here, and for instance tag your tutorial with `cancer` to make it easier to find (e.g. via the search on the website) for people who might be interested in those kinds of datasets.

e.g. like this with the covid19 tag, users can easily search across topics:

![image](https://user-images.githubusercontent.com/2563865/106020281-e1e98980-60c3-11eb-9b80-4feaa49787c7.png)
 
 you can add multiple tags if you want to as well

Reply:
sure will add them